packages/editor/vite.config.ts

```diff
@@ -18,10 +18,7 @@ export default defineConfig({
   },
   server: {
     port: 3000,
-    headers: {
-      "Cross-Origin-Embedder-Policy": "require-corp",
-      "Cross-Origin-Opener-Policy": "same-origin",
-    },
+    headers: { "Cross-Origin-Opener-Policy": "same-origin-allow-popups" },
   },
   preview: {
     headers: {
```

Comment:
the same policies probably apply for the previews (i.e. local production builds for testing) right?

Reply:
would previews be the cloudflare pages that get deployed prior to what's shown in production merge in our case?